tests/test_transforms/test_timestamp/test_fourier_transform.py

```diff
@@ -166,3 +168,44 @@ def test_forecast(ts_trend_seasonal):
 def test_save_load(ts_trend_seasonal):
     transform = FourierTransform(period=7, order=3)
     assert_transformation_equals_loaded_original(transform=transform, ts=ts_trend_seasonal)
+
+
+@pytest.mark.parametrize(
+    "transform",
+    [
+        FourierTransform(period=7, order=1),
+        FourierTransform(period=30.4, order=1),
+        FourierTransform(period=365.25, order=1),
+    ],
+)
+def test_params_to_tune(transform, ts_trend_seasonal):
+    ts = ts_trend_seasonal
+    assert len(transform.params_to_tune()) > 0
+    assert_sampling_is_valid(transform=transform, ts=ts)
+
+
+@pytest.mark.parametrize(
+    "transform",
+    [
+        FourierTransform(period=7, mods=[1]),
+        FourierTransform(period=7, mods=[1, 4]),
+    ],
+)
+def test_params_to_tune_with_mods(transform, ts_trend_seasonal):
```

Comment:
I'm not really sure does we want make this test instead of the previous one for all models, transforms.

Reply:
We discussed that it is better to return the empty grid for this case